doc/release-notes/tapyrus/release-notes-0.5.0.md

```diff
@@ -0,0 +1,71 @@
+Tapyrus version 0.5.0 is now available for download at:
+  https://github.com/chaintope/tapyrus-core/releases/tag/v0.5.0
+
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-core/issues
+
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-core/tarball/v0.5.0  # .tar.gz
+
+
+How to Upgrade
+==============
+
+If you are running a node on older version(v0.3.0, v0.4.0, v0.4.1) testnet, shut it down. Wait until it has completelyshut down. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet) to sart a new Tapyrus v0.5.0 node.
+
+If you want to running a private tapyrus network, shut down all nodes. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-new-tapyrus-network) to sart a new Tapyrus v0.5.0 network. Please note that tapyrus-signer network should also be upgraded to v0.5.0 following [tapyrus signer network setup](https://github.com/chaintope/tapyrus-signer/blob/master/doc/setup.md#how-to-set-up-new-tapyrus-signer-network)
+
```

Comment:
The introduction of the Colored Coin scheme has resulted in changes to the consensus rules. Therefore, please be reminded that all nodes need to be upgraded to 0.5.0 at the same time when upgrading.